File: benchmark/fib.py
import time

def fib(n):
    if n < 2:
        return n
    return fib(n - 1) + fib(n - 2)

print("--- PYTHON BENCHMARK (Fib 30) ---")

start = time.time()
result = fib(30)
end = time.time()

duration_ms = (end - start) * 1000

print(f"Resultat : {result}")
print(f"Temps : {duration_ms:.2f} ms")

File: benchmark/fib.rs
use std::time::Instant;

fn fib(n: u64) -> u64 {
    if n < 2 {
        n
    } else {
        fib(n - 1) + fib(n - 2)
    }
}

fn main() {
    println!("--- RUST BENCHMARK (Fib 30) ---");
    
    let start = Instant::now();
    let result = fib(30);
    let duration = start.elapsed();

    println!("Resultat : {}", result);
    println!("Temps : {} ms", duration.as_millis());
}